sdk/core/azure-core/inc/azure/core/azure_assert.hpp

```diff
@@ -0,0 +1,21 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+/**
+ * @file
+ * @brief Provide assert macros to use as pre-conditions.
+ *
+ * @remark Asserts are turned off is `NDEBUG` is defined
+ *
+ */
+
+#pragma once
+
+// uncomment next line to disable asserts
+// #define NDEBUG
+#include <cassert>
+
+#define AZURE_ASSERT(exp) assert(exp)
```

Comment:
```suggestion
#define AZURE_ASSERT(exp) std::assert(exp)
```

Reply:
assert is a MACRO, there's no namespace for it